tests/mail/tests.py

```diff
@@ -779,7 +779,8 @@ def test_send_long_lines(self):
         Message body containing longer lines are converted to Quoted-Printable
         to avoid having to insert newlines, which could be hairy to do properly.
         """
-        email = EmailMessage('Subject', "Comment ça va? " * 100, 'from@example.com', ['to@example.com'])
+        # The body length is < 998 (840) when unencoded, but > 998 when utf-8 encoded.
```

Comment:
To make the comment fit on one line: (no comma needed since what comes after isn't an independent clause)
`# Unencoded body length is < 998 (840) but > 998 when utf-8 encoded.`